packages/core/content-type-builder/admin/src/components/FormModal/utils/getAttributesToDisplay.js

```diff
@@ -21,7 +21,8 @@ const getAttributes = (dataTarget = '', targetUid, nestedComponents) => {
 
   if (isPickingAttributeForAContentType) {
     return [
-      [...defaultAttributes, 'uid'],
+      // Insert UID before the last item (richtext)
+      [...defaultAttributes.slice(0, -1), 'uid', ...defaultAttributes.slice(-1)],
```

Comment:
I'm open to suggestions to improve this bit

Reply:
I think it's fine, you could mutate the array or use a difference method but this is clear enough imo :)